content/en/blog/2023/otel-end-user-discussions-february-2023.md

```diff
@@ -0,0 +1,187 @@
+---
+title: OpenTelemetry End User Discussions Summary for February 2023
+linkTitle: End User Discussions Feb 2023
+date: 2023-02-27
+author: >-
+  [Pranay Prateek](https://github.com/pranay01) (SigNoz) <br/> with
+  contributions from: <br/> [Henrik Rexed](https://github.com/henrikrexed)
+  (Dynatrace) | [Michael Hausenblas](https://github.com/mhausenblas) (AWS) |
+  [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) | [Reese
+  Lee](https://github.com/reese-lee) (New Relic) | [Adriana
+  Villela](https://github.com/avillela) (Lightstep)
```

Comment:
Drop the `<br>` and move the "with contributions from" into the main text, as is done in https://opentelemetry.io/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql/

Reply:
Unresolving this comment since you still need to _move the phrase into the main text_ as shown in the example I cited above. Thanks.